helm_deploy/wmt-worker/values.yaml

```diff
@@ -55,10 +55,10 @@ generic-service:
       DASHBOARD_S3_ACCESS_KEY_ID: "access_key_id"
       DASHBOARD_S3_SECRET_ACCESS_KEY: "secret_access_key"
       WMT_DASHBOARD_BUCKET: "bucket_name"
-    hmpps-workload-s3-extract-sqs-output:
+    hmpps-reductions-completed-sqs-instance-output:
       ETL_SQS_ACCESS_KEY_ID: "access_key_id"
       ETL_SQS_SECRET_ACCESS_KEY: "secret_access_key"
-      ETL_SQS_QUEUE_URL: "sqs_url"
+      ETL_SQS_QUEUE_URL: "sqs_queue_url"
```

Comment:
Doesn't the etl-config.js not need changing as well?

Reply:
I think it is OK - it is using the environment variable `ETL_SQS_QUEUE_URL` which is unchanged
This file is mapping a different secret value to that same environment value